### PR TITLE
lxc: Update to 5.0.0

### PR DESCRIPTION
**Pull request by krnlyng**
[lxc] Update to 5.0.0. JB#59775

### CODE REVIEW

File: rpm/lxc.spec [hunk 1]
```diff
@@ -148,61 +149,36 @@ This package contains documentation for %{name}.
...
-sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
-sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
-
-%make_build
-
+           -Dinit-script=systemd \
+           -Dstatic=false
 
+%meson_build
+echo %{_pkgdocdir}
```
Comment (Tomin1) [inline rpm/lxc.spec]: Is this some forgotten debug logging?

Comment (krnlyng) [reply rpm/lxc.spec]: indeed thx

File: rpm/0001-Allow-building-lxc-without-init.lxc.static.patch [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
++          include_directories: liblxc_includes,
++          link_with: [liblxc_static],
++          link_args: ['-static'],
++          c_args: ['-DNO_LXC_CONF'],
++          dependencies: [libcap_static],
++          install_dir: sbindir,
++          install: true)
++  endif
+ endif
+
```
Comment (Thaodan) [inline rpm/0001-Allow-building-lxc-without-init.lxc.static.patch:71]: Upstream that patch?

**Review by Thaodan — APPROVED** [review]
LGTM

File: rpm/0001-Allow-building-lxc-without-init.lxc.static.patch (hunk 2 above)
Comment (krnlyng) [reply rpm/0001-Allow-building-lxc-without-init.lxc.static.patch:71]: Yeah i was thinking the same. But not sure if it's relevant enough to them. I'll check at some point.

**Review by mkosola — APPROVED** [review]

**Review by mlehtima — APPROVED** [review]

**Pull request merged**